src/app.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const groupBy = require('object.groupby')
+
```

Comment:
It is ok to use a library for that purpose, but I think this first challenges were most related to your ability to solve problems